aliBuild

```diff
@@ -237,10 +247,10 @@ class HttpRemoteSync:
       else:
         hashList = json.loads(urlopen(hashListUrl).read())
         pkgList = json.loads(urlopen(pkgListUrl).read())
-    except URLError, e:
+    except URLError as e:
```

Comment:
This is not compatible with python 2.4, which is currently used by slc5 (sigh). Can we factor this change out? We have in the plans to move to 2.6, but I still would like to do it separately.